src/main/com/example011/radek/v1/MinesGameApplication.java

```diff
@@ -0,0 +1,14 @@
+package main.com.example011.radek.v1;
+
+import java.security.NoSuchAlgorithmException;
+
+public class MinesGameApplication {
```

Comment:
Po spusteni se vypise hromda prazdnych radku, zrusit.

Reply:
Hotovo, bylo to pro debug při vývoji